decidim-conferences/config/locales/en.yml

```
@@ -288,7 +288,23 @@ en:
           error: There was a problem updating a registration type for this conference.
           success: Conference registration type successfully updated.
       titles:
+        conference_admins: Conference admins
+        conference_invites: Invites
+        conference_partners: Partners
+        conference_registrations: Registrations
+        conference_speakers: Conference speakers
         conferences: Conferences
+        duplicate_conference: Duplicate conference
+        media_links: Media links
+        new_conference: New conference
+        new_conference_admin: New conference admin
+        new_conference_invite: Invite participant
+        new_conference_partner: New partner
+        new_conference_speaker: New conference speaker
+        new_media_link: New media link
```

Lets change this (even though is wrong), so we have my much beloved consistency between the HTML title and the view title 😄 
```suggestion
        new_media_link: Media Link
```